projects/gloo/cli/pkg/cmd/check/prom_stats.go

```diff
@@ -0,0 +1,122 @@
+package check
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+const promStatsPath = "/stats/prometheus"
+
+func checkProxyConnectedState(stats string, genericErrMessage string, connectedStateErrMessage string) bool {
+
+	if strings.TrimSpace(stats) == "" {
+		fmt.Println(genericErrMessage+": could not find any metrics at", promStatsPath, "endpoint of the gateway-proxy deployment")
```

Comment:
nit, formatting here would be off
```suggestion
		fmt.Println(genericErrMessage+": could not find any metrics at ", promStatsPath, " endpoint of the gateway-proxy deployment")
```

also would prefer `Sprintf` here too